### DISCUSSION TITLE
Can FIO issue 1 sync for every 1 second?

### DISCUSSION THREAD

**Discussion by celestinechen** [axboe/fio#1828] [Q&A] [answered]
Hi there,

is there any option to issue one sync for every period of time? For example, issue 1 sync for every 1 second, or issue sync on a rate of 5 IOPS.

I have seen the option "fsync=X", but from the manual it looks like sending fsync for every X write commands if I understand correctly.

Thanks very much!

**Comment by vincentkfu** [comment, marked as answer]
I can think of no combination of current fio options to cleanly accomplish this. The only way I can think of to do this would be to  create an iolog and have fio use that issue operations.
